components/crud-web-apps/jupyter/backend/apps/common/utils.py

```diff
@@ -97,7 +97,7 @@ def pvc_from_dict(vol, namespace):
         spec=client.V1PersistentVolumeClaimSpec(
             access_modes=[vol["mode"]],
             storage_class_name=get_storage_class(vol),
-            resources=client.V1ResourceRequirements(requests={"storage": vol["size"]}),
+            resources=client.V1ResourceRequirements(requests={"storage": vol["size"]}), # noqa: E501
```

Comment:
instead of ignoring the line length could we do something like
```python
resources=client.V1ResourceRequirements(
    requests={"storage": vol["size"]}
),
```